src/pages/Send/components/AddressDropdownList.tsx

```diff
@@ -27,10 +28,17 @@ export const AddressDropdownList = ({
   const { network } = useNetworkContext();
   const useBtcAddress = isBitcoin(network);
   const useXPAddress = isPchainNetwork(network) || isXchainNetwork(network);
+  const useSVMAddress = Boolean(network && isSolanaNetwork(network));
 
   const selectedAddress =
     selectedContact?.[
-      useBtcAddress ? 'addressBTC' : useXPAddress ? 'addressXP' : 'address'
+      useSVMAddress
```

Comment:
how about doing something like this, since we are using the same key twice and it's getting ugly 😅 
```typescript
const addressKey = useSVMAddress
	? 'addressSVM'
	: useBtcAddress
		? 'addressBTC'
		: useXPAddress
			? 'addressXP'
			: 'address';

const selectedAddress = selectedContact?.[addressKey]?.toLowerCase();
```

Reply:
Yeah, I'll want to refactor this whole `.addressXYZ` hell (not in this PR, though).